requirements.txt

```diff
@@ -19,3 +19,5 @@ requests==2.21.0
 six==1.12.0
 urllib3==1.24.1
 whitenoise==4.1.2
+py3dns==3.2.0
+validate-email==1.3
```

Comment:
You could also remove the validate-email, and py3dns modules since you are no longer using them

Reply:
fixed please